website/docs/faqs/Environments/delete-environment-job.md

```diff
@@ -0,0 +1,57 @@
+---
+title: How to delete a job or environment in dbt Cloud?
+description: "How to delete a job or environment"
+sidebar_label: 'Delete a job or environment'
+id: delete-environment-job
+--- 
+
+
+To delete an environment or job in dbt Cloud, you must have a `developer` [license](/docs/collaborate/manage-access/seats-and-users) and have the necessary [access permissions](/docs/collaborate/manage-access/about-access). 
+
+:::info 📌 Delete a job first before deleting environment
+
+Deleting an environment doesn't automatically delete its associated job(s). If you delete an environment first without deleting the job, you won't be able to delete the job since it's without an environment. 
```

Comment:
Is there a workaround to call out here? Can support delete the job on the back end or does the job just float out in the ether for the life of the account?